src/main/java/io/axoniq/axonserver/connector/admin/AdminChannel.java

```diff
@@ -43,6 +44,21 @@
  */
 public interface AdminChannel {
 
+    /**
+     * Returns all the event processor registered to AxonServer.
+     *
+     * @return the stream of all event processor registered to AxonServer.
+     */
+    ResultStream<EventProcessor> eventProcessors();
+
+    /**
+     * Returns all the event processor registered by the specified application.
+     *
+     * @param application the application name
+     * @return the stream of all event processor registered to AxonServer by the specified application.
+     */
+    ResultStream<EventProcessor> eventProcessorsByApplication(String application);
```

Comment:
I think we use the word "component" for this in other parts of the connector. Suggest: `ResultStream<EventProcessor> eventProcessorsByComponent(String componentName);`